src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,136 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/** Class which contains the list of Task entities, and contains methods supporting the addition and removal of tasks. */
+class TaskList {
+    /** List of tasks; assume no more than 100 tasks. */
+    private ArrayList<Task> lTasks = new ArrayList<>();
+
```

Comment:
Can just name task ArrayList as 'tasks'.

Reply:
I use 'tasks' to refer to the string array which is returned by methods, and had to name this lTasks to avoid name conflict. I would be open to hearing how you named variables for your own code. Thanks!

Reply:
You can use the same name `tasks` as your attribute! To refer to it in your methods, use `this.tasks`. Research more on `this` keyword in Java if you are not familiar, you should know it :)